printer_test.go

```diff
@@ -51,6 +96,7 @@ func ExamplePrinter() {
 	//         }
 	//     }
 	// }
+
```

Comment:
@danilvpetrov It looks good to me, except for this extra new line.

Reply:
Ok, I removed that.